.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Pull Request Labeler'
+on:
+  - pull_request_target
```

Comment:
I didn't know about https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target before, seems quite useful!